oak-lucene/src/main/java/org/apache/jackrabbit/oak/plugins/index/lucene/LuceneIndexMBeanImpl.java

```diff
@@ -675,7 +690,10 @@ private static Query newDepthQuery(int depth) {
                 "numDeletedDocs",
                 "nrtIndexSize",
                 "nrtIndexSizeStr",
-                "nrtNumDocs"
+                "nrtNumDocs",
+                "lastUpdatedTimeStamp",
```

Comment:
Timestamp can be written with one or two words. Usually, in software, the single word version is used.

Reply:
done